extensions/amp-twitter/0.1/test/test-amp-twitter.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  createIframePromise,
+  doNotLoadExternalResourcesInTest,
+} from '../../../../testing/iframe';
+require('../amp-twitter');
+import {adopt} from '../../../../src/runtime';
+import {twitter} from '../../../../3p/twitter';
+
+adopt(window);
+
+describe('amp-twitter', function() {
+  this.timeout(5000);
```

Comment:
Let's drop this. We do this with flakey tests that might need more time to request external resources but since you're mocking the loading of external resources we shouldn't need these.